fedgov-cv-ISDcreator-webapp/src/main/webapp/index.html

```diff
@@ -117,7 +117,7 @@
 							<li><a href="#instructions_modal" data-toggle="modal" data-target="#instructions_modal">Instructions</a></li>
 							<li class="divider"></li>
 							<li><a href="#" onclick="alert('To request support or report problems please email\nthe CAV Support Services at: CAVSupportServices@dot.gov.');">Support</a></li>
-							<li><a href="#" data-toggle="modal" data-target="#" onclick="alert('ISD Builder Tool for J2735 3/2016 (UPER encoding)\n\n\t  Version 2.2.8, Released on 5/2/2019');">About</a></li>
+							<li><a href="#" data-toggle="modal" data-target="#" onclick="alert('ISD Builder Tool for J2735 9/2024 (UPER encoding)\n\n\t  Version 2.2.8, Released on 5/2/2019');">About</a></li>

```

Comment:
I believe we may have to update the line "Version 2.2.8, Released on 5/2/2019" to the new release version and release date.

Reply:
@DokurOmkar What is the new release version and date?